### DISCUSSION TITLE
How to monitor mimir if you do not have access to install agent crds?

### DISCUSSION THREAD

**Discussion by majerus1223** [grafana/mimir#5720] [Help and support]
How can you scrape mimir metrics if you do not have permissions to install the CRDs needed for the agent? I looked all over and cannot figure out a way to make this happen. Thanks!

**Comment by charleskorn** [comment]
Hi @majerus1223, Mimir exposes metrics at `/metrics` ([docs](https://grafana.com/docs/mimir/latest/references/http-api/#metrics)), so you'd need to manually configure either the agent or Prometheus to scrape this endpoint.
